lib/src/front_end/delimited_list_builder.dart

```diff
@@ -155,6 +154,9 @@ class DelimitedListBuilder {
 
     // See if it's an expression that supports block formatting.
     var format = switch (element) {
+      AdjacentStrings() when element.indentStrings =>
```

Comment:
```suggestion
      AdjacentStrings(indentStrings: true) =>
```

Reply:
Good idea! I'll do this locally with the other changes.

I'm so glad we decided to let object pattern fields call extensions.